conftest.py

```diff
@@ -80,33 +81,37 @@ def pytest_runtest_logstart(self, nodeid, location):
             line = self._locationline(nodeid, *location)
             self.write_ensure_prefix(line, "")
         elif self.showfspath:
-            self.write_path_name(nodeid)
-            self.write_class_name(nodeid)
+            self._write_path_name(nodeid)
+            self._write_class_name(nodeid)
+
+    def _get_formatted_test_title(self, report):
+        test_title = (report.nodeid.split("::")[-1]).split("_")
+        if test_title[0].lower() == "test":
+            test_title.pop(0)
+        test_title = " ".join(test_title)
+        return test_title
+
+    def _add_indentation_for_tests_list_item(self, report):
+        if len(report.nodeid.split("::")) >= 3:
+            return "    "
```

Comment:
It's sort of hard to tell at a glance how many spaces you want here. You could define it programmatically: `return ' '*5`.